src/OT.js

```diff
@@ -32,7 +32,7 @@ const setNativeEvents = (events) => {
   const eventNames = Object.keys(events);
   OT.setNativeEvents(eventNames);
   each(events, (eventHandler, eventType) => {
-    const allEvents = Object.keys(nativeEvents['_nativeModule']);
+    const allEvents = Object.keys(nativeEvents['_subscriber']['_subscriptionsForType']);
```

Comment:
Could you please explain a bit more what was the issue with this and how this is fixing it? Thanks!

Reply:
The `nativeEvents` object has two sub-objects on iOS `_nativeModule` and `_subscriber`, but only `_subscriber` on android. 

Looking at the events in `eventsNames` and the contents of `_nativeModule` there weren't any matches, whereas the array on '_subscriber' -> '_subscriptionsForType' had the events I was looking for. 

With hindsight, it makes sense as the JavaScript is "subscribing" to events from the native side.

Reply:
It fixes #477 using inner keys of NativeEventEmitter class. 

A more robust way would be to use the new listenerCount method

Reply:
That method seems to only return a number of events? My understanding of the original code of this function was to get a list of events and compare if the event has already been added.

Reply:
Yes.. and if a listener has been added than listenerCount is greater than 0. I opened a PR #498 to be clearer

Reply:
tested it on older versions as well and work well, couldn't cherry-pick your commit unfortunately but thanks